flowcraft-webapp/frontend/src/components/reports/phylogeny.js

```diff
@@ -40,17 +40,25 @@ export class Phylogeny extends React.Component{
 
         const metadata = new Map();
 
-        for (const m of rawMetadata){
-            const currentMeta = m.reportJson.metadata[0];
-            const sample = currentMeta.sample;
+        for (const m of rawMetadata) {
```

Comment:
What is `m`? try to make variables more intuitive

Reply:
resolved in the latest commit